src/App.js

```diff
@@ -37,55 +61,79 @@ function App() {
         setIsLoading(false);
       });
     }
-
     return unsubscribe;
   }, [token]);
 
   return (
-    <div className="App">
-      <header className="App-header">
-        <div style={{ background: '#fff', padding: '40px', borderRadius: 5 }}>
-          <Switch>
-            <Route
-              exact
-              path="/"
-              render={() =>
-                token ? (
-                  <Redirect to="/list" />
-                ) : (
-                  <Welcome setToken={setToken} />
-                )
-              }
+    <Box className="App">
+      <Grid container justify="center">
+        {/* <Grid item xs={12} sm={6} md={4} style={{height: "90vh", marginTop: "5vh"}}> more responsive */}
```

Comment:
Are you saving this for after the demo? Or can delete?

Reply:
Saving it for after the demo!